gridstatus/ercot.py

```diff
@@ -2029,37 +2024,231 @@ def _handle_sced_timestamp(self, df, verbose=False):
 
         return df
 
-    def _handle_sced_system_lambda(self, docs, verbose):
-        all_dfs = []
-        for doc in tqdm.tqdm(
+    @support_date_range("DAY_START")
+    def get_shadow_prices_dam(self, date, end=None, verbose=False):
+        """Get Day-Ahead Market Shadow Prices
+
+        Arguments:
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame with day-ahead market shadow prices
+        """
+        # Subtract one day since this is the day ahead market
+        date = date if date == "latest" else date - pd.DateOffset(days=1)
+
+        doc = self._get_document(
+            date=date,
+            report_type_id=DAM_SHADOW_PRICES_RTID,
+            verbose=verbose,
+        )
+
+        return self._handle_shadow_prices_dam_doc(doc, verbose=verbose)
+
+    def _handle_shadow_prices_dam_doc(self, doc, verbose):
+        # Use skipinitialspace to remove spaces after delimiters in all cells
+        df = self.read_doc(
+            doc,
+            parse=True,
+            verbose=verbose,
+            read_csv_kwargs=dict(skipinitialspace=True),
```

Comment:
`skipinitialspace` removes leading spaces in column names and cell values. I think we could use this in other places, but I didn't want to change the default behavior, so I used it only for the `shadow_prices` methods.